src/templates/blog-post.tsx

```diff
@@ -20,6 +20,9 @@ const BlogArticleTemplate: React.FC<BlogArticleTemplateProps> = ({
   const { t } = useTranslation();
   const markdown = data.markdownRemark;
 
+  const shareImagePath =
+    markdown.frontmatter.shareImage.childImageSharp.fixed.src;
```

Comment:
we switch from card to the image for blog posts here. The shareImage is just another variant of the teaser cropped to the expected format.